src/main/java/frc/robot/Robot.java

```diff
@@ -69,15 +72,15 @@ public void disabledExit() {
 
     @Override
     public void autonomousInit() {
-
-        compressor.disable();
         autonomousCommand = autonSelector.chooseAuton();
+        autonomousDelay = delaySelector.chooseDelay();
         // if (DriverStation.isFMSAttached()) {
         // vision.startRecording();
         // }
 
         if (autonomousCommand != null) {
             driveTrain.setDrivingIdleMode(SwerveModuleConstants.kAutoDrivingMotorIdleMode);
+            Timer.delay(autonomousDelay);
```

Comment:
I still believe that adding a WaitCommand into the sequential command group inside of the Auton abstract class is a better implementation

Reply:
I just thought this way would be nicer because if someone wanted to make a new auton they wouldn't have to remeber to put a waitCommand at the top of the class especially if it was during a comp and it just slipped their minds

Reply:
If you put this in the auton class that all autons extend from then it won’t be an issue